docs/DeveloperGuide.adoc

```diff
@@ -840,7 +875,82 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. NUSCouples shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Add Timetable to Person
+
+*MSS*
+
+1.  User requests to list persons.
+2.  NUSCouples shows a list of persons.
+3.  User request to add a shortened http://nusmods.com[NUSMODS] timetable url to a specific person in the list.
+4.  NUSCouples adds the url to the person in the list.
+5.  NUSCouples displays the timetable.
```

Comment:
No request from the user? may be you can merge this to the previous step. Usually, the steps in MSS show interaction between user and the system. So, having two consecutive user/system steps are not recommended..